docs-react-native/react-native/docs/ios/remote-notification-support.md

```diff
@@ -64,7 +45,7 @@ target 'NotifeeNotificationService' do
 end
 
 ```
-
+* Assuming you are using `react-native-firebase`, you'll also have to add `use_frameworks!` inside the target above as [you did when you followed their docs](https://rnfirebase.io/#altering-cocoapods-to-use-frameworks). 
```

Comment:
Avoid contractions in documentation where possible (I try to remove them when I notice them)
This may help people with relatively weaker English language skills, I am sympathetic to that

```suggestion
* Assuming you are using `react-native-firebase`, you must also have to add `use_frameworks!` inside the target above as [you did when you followed their docs](https://rnfirebase.io/#altering-cocoapods-to-use-frameworks). 
```